src/Tipsy.js

```diff
@@ -11,6 +10,27 @@ import PropTypes from 'prop-types';
  * Like bootstrap's tooltip plugin, Tipsy does not rely on images.
  */
 
+class Container extends PureComponent {
```

Comment:
What's the reason for this being a PureComponent instead of a stateless functional component?

Reply:
The version of `ReactTispy` in this pull request branch uses `unstable_renderSubtreeIntoContainer` method to open a React Subtree into the major DOM outside of the React Managed vDOM.  Which (at least at the time) didn't support `Functional` components

Reply:
Why `PureComponent` over `Component`?

Unless we've measured performance, or think that the props will not change very much, I'm not sure we want to jump to the conclusion that it will be better, @JustinRyanH

Reply:
No specific reason for PureComponent over Component. I thought that since it didn't have lifecycles it didn't need to be a Component.